components/centraldashboard/webpack.config.js

```diff
@@ -1,20 +1,20 @@
 'use strict';
 
-const { resolve } = require('path');
-const CleanWebpackPlugin = require('clean-webpack-plugin');
-const CopyWebpackPlugin = require('copy-webpack-plugin');
-const DefinePlugin = require('webpack').DefinePlugin;
-const HtmlWebpackPlugin = require('html-webpack-plugin');
-const MiniCssExtractPlugin = require("mini-css-extract-plugin");
-const ScriptExtHtmlWebpackPlugin = require('script-ext-html-webpack-plugin');
+const {resolve} = require('path')
+const CleanWebpackPlugin = require('clean-webpack-plugin')
+const CopyWebpackPlugin = require('copy-webpack-plugin')
+const DefinePlugin = require('webpack').DefinePlugin
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const MiniCssExtractPlugin = require("mini-css-extract-plugin")
+const ScriptExtHtmlWebpackPlugin = require('script-ext-html-webpack-plugin')
 
-const NODE_MODULES = /\/node_modules\//;
-const ENV = process.env.NODE_ENV || 'development';
-const PKG_VERSION = require('./package.json').version;
-const SRC = resolve(__dirname, 'public');
-const COMPONENTS = resolve(SRC, 'components');
-const DESTINATION = resolve(__dirname, 'dist', 'public');
-const WEBCOMPONENTS = './node_modules/@webcomponents/webcomponentsjs';
+const ENV           = process.env.NODE_ENV || 'development'
```

Comment:
This would be discouraged by the Google style guide.

https://google.github.io/styleguide/jsguide.html#formatting-horizontal-alignment